tools/generator/src/Generator/CreateXcodeProj.swift

```diff
@@ -4,12 +4,14 @@ extension Generator {
     /// Creates an `XcodeProj` for the given `PBXProj`.
     static func createXcodeProj(
         for pbxProj: PBXProj,
-        sharedData: XCSharedData?
+        sharedData: XCSharedData?,
+        userData: [XCUserData]
```

Comment:
Until we need this to be an array, we should accept a single value here, and wrap it in an array in the `XcodeProj` call below.
```suggestion
        userData: XCUserData
```

Reply:
Sounds good! Changed to wrapping `XCUserData` in `XcodeProj` call everywhere https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2151/commits/dbea5e45c39c8826c2951ce1799fc5aac7cc6d84